content/docs/get-started/quickstart.mdx

```diff
@@ -104,24 +104,24 @@ Pomerium will deploy and run in a special "Zero-managed" mode. In Zero-managed m
 
 If your Pomerium deployment connects to the cloud successfully, Pomerium Zero will notify you in the onboarding screen:
 
-![The successful connection notification in the Pomerium Zero onboarding flow](./img/quickstart/zero-quickstart-connection.png)
+![The successful connection notification in the Pomerium Zero onboarding flow](./quickstart/zero-quickstart-connection.png)
 
 Once you've connected, select **Finish**. You'll be directed to the Zero Console.
 
 :::tip Troubleshoot connection issues
 
 If you can't connect to Pomerium Zero, check out these pages for help:
 
-- [**Configure port 443 to allow inbound access**](/docs/troubleshooting#configure-port-443-to-allow-inbound-access)
```

Comment:
Is there a reason to remove the anchor that takes visitors directly to that section of the troubleshooting page?